docs-gitbook/quickstarts/deploy-a-subnet.md

```diff
@@ -117,11 +126,47 @@ gateway_addr = "<GATEWAY_ADDR>"
 registry_addr = "<REGISTRY_ADDR>"
 ```
 
-*   **Replace** the `gateway_addr` and `registry_addr` with the following values. Click on the badges below to take you to the source to copy and paste them or go to [this link](https://github.com/consensus-shipyard/ipc/blob/cd/contracts/deployments/r314159.json).
+- **Replace** the `gateway_addr` and `registry_addr` with the following values. Click on the badges below to take you to the source to copy and paste them or go to [this link](https://github.com/consensus-shipyard/ipc/blob/cd/contracts/deployments/r314159.json).
+
+  [![Gateway Address](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Fconsensus-shipyard%2Fipc%2Fcd%2Fcontracts%2Fdeployments%2Fr314159.json&query=%24.gateway_addr&label=Gateway%20Address)](https://github.com/consensus-shipyard/ipc/blob/cd/contracts/deployments/r314159.json)
+
+  [![Registry Address](https://img.shields.io/badge/dynamic/json?url=https%3A%2F%2Fraw.githubusercontent.com%2Fconsensus-shipyard%2Fipc%2Fcd%2Fcontracts%2Fdeployments%2Fr314159.json&query=%24.registry_addr&label=Registry%20Address)](https://github.com/consensus-shipyard/ipc/blob/cd/contracts/deployments/r314159.json)
+
+If you want to deploy custom ipc subnet on mainnet, you will need your own ipc stack of contracts deployed.
+
+```bash
+cd contracts
+```
+
+- Populate `.env` file with your `PRIVATE_KEY` and `RPC_URL`.
+- Set your `NETWORK` variable to `mainnet` value.
+
+```
+export NETWORK = mainnet
```

Comment:
why mainnet, from the description, I think it's trying to deploy to calibration?